idaes/models/unit_models/packed_bed_reactor.py

```diff
@@ -0,0 +1,337 @@
+        doc="""Property parameter object used to define property calculations,
+**default** - useDefault.
+**Valid values:** {
+**useDefault** - use default package from parent model or flowsheet,
+**PropertyParameterObject** - a PropertyParameterBlock object.}"""))
+    CONFIG.declare("property_package_args", ConfigBlock(
+        implicit=True,
+        description="Arguments to use for constructing property packages",
+        doc="""A ConfigBlock with arguments to be passed to a property block(s)
+and used when constructing these,
+**default** - None.
+**Valid values:** {
+see property package for documentation.}"""))
+    CONFIG.declare("reaction_package", ConfigValue(
+        default=None,
+        domain=is_reaction_parameter_block,
+        description="Reaction package to use for control volume",
+        doc="""Reaction parameter object used to define reaction calculations,
+**default** - None.
+**Valid values:** {
+**None** - no reaction package,
+**ReactionParameterBlock** - a ReactionParameterBlock object.}"""))
+    CONFIG.declare("reaction_package_args", ConfigBlock(
+        implicit=True,
+        description="Arguments to use for constructing reaction packages",
+        doc="""A ConfigBlock with arguments to be passed to a reaction block(s)
+and used when constructing these,
+**default** - None.
+**Valid values:** {
+see reaction package for documentation.}"""))
+    CONFIG.declare("length_domain_set", ConfigValue(
+        default=[0.0, 1.0],
+        domain=ListOf(float),
+        description="List of points to use to initialize length domain",
+        doc="""A list of values to be used when constructing the length domain
+of the reactor. Point must lie between 0.0 and 1.0,
+**default** - [0.0, 1.0].
+**Valid values:** {
+a ListOf(float)}"""))
+    CONFIG.declare("transformation_method", ConfigValue(
+        default="dae.finite_difference",
+        description="Method to use for DAE transformation",
+        doc="""Method to use to transform domain. Must be a method recognised
+by the Pyomo TransformationFactory,
+**default** - "dae.finite_difference"."""))
+    CONFIG.declare("transformation_scheme", ConfigValue(
+        default="BACKWARD",
+        description="Scheme to use for DAE transformation",
+        doc="""Scheme to use when transformating domain. See Pyomo
+documentation for supported schemes,
+**default** - "BACKWARD"."""))
+    CONFIG.declare("finite_elements", ConfigValue(
+        default=20,
+        description="Number of finite elements to use for DAE transformation",
+        doc="""Number of finite elements to use when transforming length
+domain,
+**default** - 20."""))
+    CONFIG.declare("collocation_points", ConfigValue(
+        default=3,
+        description="No. collocation points to use for DAE transformation",
+        doc="""Number of collocation points to use when transforming length
+domain,
+**default** - 3."""))
+    CONFIG.declare("mass_catalyst", ConfigValue(
```

Comment:
I do not think these are things that should be set as config arguments - without having seen the following code these look like they are just default values for parameters that will be constructed later.

For cases like this, we prefer to get the user to set the values directly after the model is constructed.